Backpack-SwiftUI/Switch/Classes/BPKSwitch.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKSwitch: View {
+    @Binding private var isOn: Bool
+    
+    public init(isOn: Binding<Bool>) {
+        self._isOn = isOn
+    }
+    
+    public var body: some View {
+        Toggle(isOn: $isOn) {
+            Group {}
+        }
+        .labelsHidden()
```

Comment:
one thing we've done on android (for all toggleable controls) is provide a version with the label (with our text style) and then another which adds a content slot. 
the main reason for that is that on android you generally tap the whole row, rather than just the toggle to switch mode and it allows us to set accessibility rules on the whole content, rather than just the toggle. not sure if that's similar on iOS or if you generally only tap the switch to toggle

Reply:
Yes, actually the native Toggle (Switch) on SwiftUI has a Label to treat it as a row. I did it this way to make it work same as it was on UIKit, but adding a version that can receive a label (or any other view) may be a good idea, since it's the way it will actually be used